src/vs/loader.js

```diff
@@ -789,7 +789,12 @@ var AMDLoader;
                 catch (_e) {
                     recorder.record(61 /* CachedDataMissed */, cachedDataPath);
                 }
-                var script = new that._vm.Script(scriptSource, options);
+                var script;
```

Comment:
sql carbon edit comments?

Reply:
Oh right, let me wait and see if they accept my PR first.

Reply:
No word from them yet so just going to get this in and we can remove it later if they accept the change.